README.md

```diff
@@ -52,7 +52,14 @@
         prefix: '/%sylius_admin.path_name%'
     ```
 
-3. Clear cache
+3. Import twig hooks from config in your `config/packages/maintenance.yaml` file:
```

Comment:
We can put it on the [recip contrib](https://github.com/symfony/recipes-contrib/tree/main/synolia) in the future to simplify installation.